File: ORB.Core/Models/PartnerOrganisationRiskRiskType.cs
using System;
using System.Collections.Generic;

namespace ORB.Core.Models
{
    public partial class PartnerOrganisationRiskRiskType : EntityWithEditor
    {
        public int PartnerOrganisationRiskID { get; set; }
        public int RiskTypeID { get; set; }

        public virtual PartnerOrganisationRisk PartnerOrganisationRisk { get; set; }
        public virtual RiskType RiskType { get; set; }
    }
}

File: corporate-reporting/src/types/PartnerOrganisationRiskRiskType.ts
import { IEntity, Entity } from "./Entity";

export interface IPartnerOrganisationRiskRiskType extends IEntity {
    PartnerOrganisationRiskID: number;
    RiskTypeID: number;
}

export class PartnerOrganisationRiskRiskType extends Entity implements IPartnerOrganisationRiskRiskType {
    public PartnerOrganisationRiskID = null;
    public RiskTypeID = null;
}